src/main/java/lotto/domain/Amount.java

```diff
@@ -0,0 +1,59 @@
+package lotto.domain;
+
+import lotto.util.Parser;
+
+public class Amount {
+	private final long amount;
+
+	public Amount(long amount) {
+		this.amount = amount;
+	}
+
+	public Amount(String amount) {
+		this.amount = Parser.toLong(amount);
```

Comment:
```suggestion
		this(Parser.toLong(amount));
```

기준이 되는 생성자를 하나 정해보고 this 형태로 호출해 보는것은 어떨까요?! 🤔